src/components/TeamPics/teampics.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import TRUMAN from '../../assets/images/truman.jpg';
+import LORENZO from '../../assets/images/lorenzo.jpg';
+import MARY from '../../assets/images/mary.jpg';
+//import SHIV from '../../assets/images/shiv.jpg';
+//import ATHENA from '../../assets/images/athena.jpg';
+//import ADRIAN from '../../assets/images/adrian.jpg';
+//import BUSHER from '../../assets/images/busher.jpg';
+//import TUZETTE from '../../assets/images/tuzette.jpg';
+//import MANJOT from '../../assets/images/manjot.jpg';
+//import KACEY from '../../assets/images/kacey.jpg';
+//import SID from '../../assets/images/sid.jpg';
+//import INTISER from '../../assets/images/intiser.jpg';
+
+import './teampics.scss';
+
+
+/* Outline for this component largely taken from: https://www.codesmite.com/article/how-to-create-pure-css-hexagonal-grids */
+
+
+const Test: React.FC = () => {
```

Comment:
Can this be changed to the name of the component instead of `Test`

Reply:
Oh! I missed that. Is it possible for me to push this change, and have @snakethebigboss pull the new changes while keeping his current changes? Or should he himself have this as one of his fixes so everything's more smoother?

Reply:
pull down the `feat/about-us` branch and make your changes on this branch then push up